proto/cosmos/accesscontrol/constants.proto

```diff
@@ -0,0 +1,28 @@
+syntax = "proto3";
+package cosmos.accesscontrol.v1beta1;
+
+option go_package = "github.com/cosmos/cosmos-sdk/x/accesscontrol/types";
+
+enum ResourceType {
+    ANY = 0;
+    KV = 1;
+    Mem = 2;
+    DexMem = 3;
+}
+
+enum MessageType {
+    ANY_MESSAGE = 0;
+    // TODO(bweng/uday):: Add more message type support
+}
+
+enum Module {
```

Comment:
for module we probably want to just use a string, since `sei-cosmos` should not have knowledge of custom modules in sei

Reply:
Yessir, updated to use a string